project/src/const.tsx

```diff
@@ -14,3 +14,9 @@ export enum Titles {
   FavoriteTitle = 'Saved listing',
   YourReview = 'Your review',
 }
+
+export const URL_MARKER_DEFAULT = 'img/pin.svg';
+
+export const URL_MARKER_ACTIVE = 'img/pin-active.svg';
+
+export const CITIES_LIST = ['Paris', 'Cologne', 'Brussels', 'Amsterdam', 'Hamburg', 'Dusseldorf'];
```

Comment:
```suggestion
export const CITIES_LIST = ['Paris', 'Cologne', 'Brussels', 'Amsterdam', 'Hamburg', 'Dusseldorf'] as const;
```
попробуй навести мышкой - он тебе сразу посдкажет - что элемент массива это не просто строка, а один из городов